confignode/src/main/java/org/apache/iotdb/confignode/manager/cq/CQScheduleTask.java

```diff
@@ -216,6 +204,18 @@ public AsyncExecuteCQCallback(long startTime, long endTime) {
       this.endTime = endTime;
     }
 
+    private void updateExecutionTime() {
+      if (timeoutPolicy == TimeoutPolicy.BLOCKED) {
+        executionTime = executionTime + everyInterval;
+      } else if (timeoutPolicy == TimeoutPolicy.DISCARD) {
+        long now = System.currentTimeMillis();
+        executionTime =
+            executionTime + ((now - executionTime - 1) / everyInterval + 1) * everyInterval;
+      } else {
+        throw new IllegalArgumentException("Unknown TimeoutPolicy: " + timeoutPolicy);
+      }
+    }
+
```

Comment:
Why move these method?

Reply:
"private" methods called only by inner classes should be moved to those classes

Reply:
Get it!